### PR TITLE
EpicGames,Inc.EpicGamesLauncher version 1.2.17.0

### PR DESCRIPTION
**Pull request by UselessEktaChele**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16170)

### CODE REVIEW

File: manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: EpicGames,Inc.EpicGamesLauncher
```
Comment (jedieaston) [inline manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml:3]: This is a duplicate of `EpicGames.EpicGamesLauncher`.

**Review by jedieaston — CHANGES_REQUESTED** [review]
This is a duplicate of `EpicGames.EpicGamesLauncher`, and as such shouldn't be merged.

Next time, try using `winget search` to see if a manifest already exists in the repo. I got a result for Epic Games Launcher:

```
PS C:\Users\Easton Pillay\projects\winget-pkgs> winget search epicgameslauncher
Name                Id                          Version  Source
----------------------------------------------------------------
Epic Games Launcher EpicGames.EpicGamesLauncher 1.2.17.0 winget
PS C:\Users\Easton Pillay\projects\winget-pkgs> 
```

File: manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml (hunk 1 above)
Comment (TechPhoenix-89) [reply manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml:3]: bruh i searched, and it wasn't there, what should i do, tell microsoft, or anyone, who knows how to fix that.

Comment (OfficialEsco) [reply manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml:3]: Could you post a screenshot of your search? What you can do is add tags so its easier to find.

Comment (TechPhoenix-89) [reply manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml:3]: ok found it, i searched games store, and it is supposed to be games launcher, so yeah i am sorry for the rude behaviour.

Comment (OfficialEsco) [reply manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml:3]: Tell me the tags you would like to find the Epic Games Store by and ill add them once i get around to it, the moniker is egs so you might be right that it should be Epic Games Store, however Epic Games Launcher seems like the name it has in Windows unless its incorrect.

Comment (OfficialEsco) [reply manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml:3]: Fixed in #16208, you can add more tags if need be.

Comment (TechPhoenix-89) [reply manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml:3]: how do you add tags?

**Pull request closed without merging**